tests/common.h.in

```diff
@@ -39,35 +37,20 @@ namespace geode
  * from the context in which OPENGEODE_TEST is called.
  */
 
-#ifdef OPENGEODE_BENCHMARK
-#    define CATCH_CONFIG_MAIN
-#    define CATCH_CONFIG_PREFIX_ALL
-#    define CATCH_CONFIG_ENABLE_BENCHMARKING
-#    include <geode/tests/catch2.h>
-#    define OPENGEODE_TEST( name )                                             \
-        CATCH_TEST_CASE( name )                                                \
-        {                                                                      \
-            CATCH_BENCHMARK( name )                                            \
-            {                                                                  \
-                return test();                                                 \
-            };                                                                 \
-        }
-#else
-#    define OPENGEODE_TEST( name )                                             \
-        int main()                                                             \
-        {                                                                      \
-            using namespace geode;                                             \
+#define OPENGEODE_TEST( name )                                                 \
+    int main()                                                                 \
+    {                                                                          \
+        using namespace geode;                                                 \
                                                                                \
-            try                                                                \
-            {                                                                  \
-                test();                                                        \
+        try                                                                    \
```

Comment:
is it still necessary ?

Reply:
No but no need to port all the current tests either..